sbndqm/Decode/PMT/SBND/DaqDecoderSBNDPMT_module.cc

```diff
@@ -0,0 +1,302 @@
+////////////////////////////////////////////////////////////////////////
+// Class:       DaqDecoderSBND
+// Plugin Type: producer (art v2_09_06)
+// File:        DaqDecoderSBND.cxx
+//
+// Lan Nguyen
+// Email: vclnguyen1@sheffield.ac.uk
+////////////////////////////////////////////////////////////////////////
+
+#include "art/Framework/Core/EDProducer.h"
+#include "art/Framework/Principal/Event.h"
+#include "art/Framework/Principal/Handle.h"
+#include "canvas/Utilities/InputTag.h"
+#include "messagefacility/MessageLogger/MessageLogger.h"
+#include "cetlib_except/exception.h"
+#include "canvas/Utilities/InputTag.h"
+
+#include "fhiclcpp/ParameterSet.h"
+#include "fhiclcpp/types/TableAs.h"
+#include "fhiclcpp/types/Sequence.h"
+#include "fhiclcpp/types/OptionalAtom.h"
+#include "fhiclcpp/types/Atom.h"
```

Comment:
It looks like this is the only one you are using in the code:
```suggestion
#include "fhiclcpp/types/Sequence.h"
```

Reply:
solved with 6b5231e